storm-core/src/native/worker-launcher/impl/worker-launcher.c

```diff
@@ -485,14 +482,14 @@ int setup_stormdist_dir(const char* local_dir) {
 
       case FTS_DP:        // A directory being visited in post-order
       case FTS_DOT:       // A dot directory
+      case FTS_SL:        // A symbolic link
+      case FTS_SLNONE:    // A broken symbolic link
```

Comment:
I talked with @tgravescs and ran some tests.  It works just fine, and it should not be a big deal.  I was a bit worried about the default permissions on the symlinks but they are open, and because they are all under a directory that should be locked down I am not concerned abut it.